tests/base.py

```diff
@@ -61,32 +61,23 @@ def get_credentials(self):
             'access_token': os.getenv("TAP_GITHUB_TOKEN")
         }
 
-    @staticmethod
-    def expected_check_streams():
-        return {
-            'assignees',
-            'collaborators',
-            'comments',
-            'commit_comments',
-            'commits',
-            'events',
-            'issue_labels',
-            'issue_milestones',
-            'issue_events',
-            'issues',
-            'pr_commits',
+    # BUG TDL-16571
+    # The folowing streams are not honoring automatic field selection
+    def expected_check_streams(self):
```

Comment:
This method is intended to return all streams that are discovered. If they are not being discovered this issue should be fixed or documented. The comment above suggests that they fail in a specific test. In which case they should be skipped in that test, not in the base.py method. The only exception to this is if the stream does not pass any of the tests.

Reply:
Also it should just return the keys from expected_metadata.

Reply:
Removed from base as bug was identified in tap code